source/server/server.cc

```diff
@@ -644,9 +644,12 @@ RunHelper::RunHelper(Instance& instance, const Options& options, Event::Dispatch
       }) {
   // Setup signals.
   if (options.signalHandlingEnabled()) {
-// TODO(Pivotal): Figure out solution to graceful shutdown on Windows. None of these signals exist
-// on Windows.
-#ifndef WIN32
+#ifdef WIN32
+    sigterm_ = dispatcher.listenForSignal(ENVOY_WIN32_SIGTERM, [&instance]() {
```

Comment:
Can we make the SIGTERM handler for WIN32 and non-WIN32 the same? Maybe define `ENVOY_SIGTERM` on all platforms and use that?

Reply:
Will do that

Reply:
fixed